src/site-server.ts

```diff
@@ -70,11 +70,19 @@ function getAbsoluteUrl( details: SiteDetails ): string {
 	return `http://localhost:${ details.port }`;
 }
 
+// We use SiteDetails for storing it in appdata-v1.json, so this meta was introduced for extra data which is not stored locally
+type SiteServerMeta = {
+	wpVersion?: string;
```

Comment:
@sejas This is only one point where I am not sure that it's the best approach. I am not very familiar with wpnow etc, so could you please take a look on my approach to say - is it good, or maybe it's better to go another way.